adr/2022-08-decouple-from-ietf.md

```diff
@@ -52,31 +52,36 @@ from the IETF process.
 
 1. Continue to submit I-Ds, while using our customized process with no intention
   of pursing standards track RFC status.
-2. Get JSON Schema to a stable place and pursue a standards track RFC with the
-  IETF.
-3. Decouple from IETF completely and come up with our own governance model.
+2. Go all-in with IETF and pursue a standards track RFC with the IETF.
+3. Join W3C and pursue a standards track with them using their process.
+4. Decouple completely from any standards organization and come up with our own
+   specification development lifecycle (SDLC) model inspired by well established
+   projects with an SDLC that more closely meets or needs.
 
 ## Decision Outcome
 
-Our decision is to go with Option 3 and decouple from IETF completely and come
-up with our own governance model. The idea of having to come up with our own
-governance model is daunting, but when the alternatives are to continue to abuse
-the I-D process or to conform to the IETF process that doesn't fit our needs,
-going our own way seems the best option.
+Our decision is to go with Option 4 and decouple from standards organizations
+that don't fit our needs. We don't currently have a plan for what to replace
+IETF with. We are currently investigating how other established projects do
+their SDLC and will likely choose one to emulate and adapt to our needs.
```

Comment:
I think it would be good to identify a few written SDLC processes (as opposed to just pointing to projects and saying "what they're doing", although some SDLC processes may be project-specific documents and still be worth referencing) that we are considering.  That would make this feel a little less like we're just kind of wandering off indefinitely.

I do not have links for SDLCs off the top of my head, I figure I'd see what the reaction to this idea was before doing any research.